en/docs/install-and-setup/setup/mi-setup/deployment/mi-cicd-overview.md

```diff
@@ -0,0 +1,31 @@
+# Building a Jenkins CI/CD Pipeline for Micro Integrator
+
+## Overview
+
+This is a guide of a reference implementation on setting up a CI/CD process for Micro Integrator. This guide contains two parts.
```

Comment:
Can we add some information on when someone would do this or in what exact scenario? It's better to have that kind of content in the overview.

Reply:
Added some info. Please review